README.md

```diff
@@ -58,8 +58,8 @@ CHANGELOG
 * API change: WildMidi_GetError() and WildMidi_ClearError() added to
   cleanly check for, retrieve and clear error messages. They no longer
   go to stderr.
-* Support for loading XMI (XMIDI format) files, such as from Arena. Thanks
-  Ryan Nunn for releasing his code under the LGPL.
+* Support for loading XMI (XMIDI format) and XFM files, such as from Arena.
+  Thanks Ryan Nunn for releasing his code under the LGPL.
```

Comment:
Ryan Nunn was the inspiration for WildMIDI support of XMI and thanks to him we have a xmi2midi converter, but the native player used in WildMIDI was created by the WildMIDI team (Chris Ison, Sezero and myself). :)